hack/update-copyright.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
```

Comment:
can you open a ticket that will be part of our `make update` command?
and open another ticket to add to our CI that we will add a [check generated files](https://github.com/IBM/ibm-block-csi-operator/blob/develop/Makefile#L79-L80) step that will verify for each branch that we won't miss copyrights??

Reply:
Opened CSI-5233 and CSI-5234

Reply:
thanks